src/PolygonExtractor.js

```diff
@@ -105,6 +105,10 @@ PolygonExtractor.prototype.doneHandler = function (/*results*/) {
 PolygonExtractor.prototype.cleanAdminLevel = function (adminLevel) {
   var adminLevelReg = new RegExp(/^\d+$/);
 
+  if ((adminLevel == undefined) || (adminLevel == null)){
```

Comment:
we're pretty consistent with `===` throughout the code, so would be great if you could update to using that in this PR.